dojo/test/signals.py

```diff
@@ -1,13 +1,39 @@
 import contextlib
-from django.db.models import signals
+from auditlog.models import LogEntry
+from django.conf import settings
+from django.contrib.contenttypes.models import ContentType
 from django.dispatch import receiver
-import logging
+from django.db.models.signals import pre_save, post_delete
+from django.utils.translation import gettext as _
+from django.urls import reverse
+
 from dojo.models import Test, Finding
+from dojo.notifications.helper import create_notification
+
 
-logger = logging.getLogger(__name__)
+@receiver(post_delete, sender=Test)
+def test_post_delete(sender, instance, using, origin, **kwargs):
```

Comment:
No notifications are generated when deleting a Test by either UI or API (apologies for the confusing names used here - Product name=`test1`, Engagement name=`test`, Test name=`test1` 😅 )

![Screenshot 2024-04-09 at 20 45 00](https://github.com/DefectDojo/django-DefectDojo/assets/1749665/67f1955d-d2f9-45ad-bcff-446ea1538720)